support/ci/should_we_build.sh

```diff
@@ -14,6 +14,12 @@ echo "======================================================="
 echo "Testing whether or not we need to build ${package_name}"
 echo "======================================================="
 
+hab pkg install core/rq
+hab pkg binlink core/rq rq
+
+hab pkg install core/jq-static
+hab pkg binlink core/jq-static jq
+
```

Comment:
@mwrock Why did you need to add these? They should already be installed in `.travis.yml` (`jq` from Ubuntu packages, and `rq` from the [install_rq.sh](https://github.com/habitat-sh/habitat/blob/master/support/ci/install_rq.sh) script.

Also, this script is going to be run multiple times (once per package in the build), so we're going to be unnecessarily trying to install two Habitat packages each time.

Reply:
oh i can take that out. I added it because I hit issues locally (not in travis) but thats not really the use case for the deploy script.